pkg/clientutil/predicates.go

```diff
@@ -53,6 +53,16 @@ func PredicateClusterByAccessMode(accessMode greenhousev1alpha1.ClusterAccessMod
 	})
 }
 
+func PredicateClusterIsReady() predicate.Predicate {
+	return predicate.NewPredicateFuncs(func(o client.Object) bool {
+		cluster, ok := o.(*greenhousev1alpha1.Cluster)
+		if !ok {
+			return false
+		}
+		return len(cluster.Status.Conditions) > 0 && cluster.Status.IsReadyTrue()
```

Comment:
`cluster.Status.IsReadyTrue()` might cause a nil pointer, right? 
If the [IsReadyTrue](https://github.com/cloudoperators/greenhouse/blob/main/pkg/apis/greenhouse/v1alpha1/conditions.go#L151) is executed on a `nil` ReadyCondition.
But this should be fixed generally on the Condition?

Reply:
I provided a fix with #639